modules/sovrnBidAdapter.js

```diff
@@ -1,156 +1,83 @@
-      missing = [];
-    }
-
-    missing.forEach(function (bidRequest) {
-      // Add a no-bid response for this bid request.
-      var bid = {};
-      bid = bidfactory.createBid(2, bidRequest);
-      bid.bidderCode = 'sovrn';
-      bidmanager.addBidResponse(bidRequest.placementCode, bid);
-    });
-  }
-
-  // expose the callback to the global object:
-  $$PREBID_GLOBAL$$.sovrnResponse = function (sovrnResponseObj) {
-    var impidsWithBidBack = [];
-
-    // valid response object from sovrn
-    if (sovrnResponseObj && sovrnResponseObj.id && sovrnResponseObj.seatbid && sovrnResponseObj.seatbid.length !== 0 &&
-      sovrnResponseObj.seatbid[0].bid && sovrnResponseObj.seatbid[0].bid.length !== 0) {
-      sovrnResponseObj.seatbid[0].bid.forEach(function (sovrnBid) {
-        var responseCPM;
-        var placementCode = '';
-        var id = sovrnBid.impid;
-        var bid = {};
-
-        var bidObj = utils.getBidRequest(id);
-
-        if (bidObj) {
-          placementCode = bidObj.placementCode;
-          bidObj.status = CONSTANTS.STATUS.GOOD;
-
-          responseCPM = parseFloat(sovrnBid.price);
-
-          if (responseCPM !== 0) {
-            sovrnBid.placementCode = placementCode;
-            sovrnBid.size = bidObj.sizes;
-            var responseAd = sovrnBid.adm;
-
-            // build impression url from response
-            var responseNurl = '<img src="' + sovrnBid.nurl + '">';
-
-            // store bid response
-            // bid status is good (indicating 1)
-            bid = bidfactory.createBid(1, bidObj);
-            bid.creative_id = sovrnBid.id;
-            bid.bidderCode = 'sovrn';
-            bid.cpm = responseCPM;
-
-            // set ad content + impression url
-            // sovrn returns <script> block, so use bid.ad, not bid.adurl
-            bid.ad = decodeURIComponent(responseAd + responseNurl);
-
-            // Set width and height from response now
-            bid.width = parseInt(sovrnBid.w);
-            bid.height = parseInt(sovrnBid.h);
-
-            if (sovrnBid.dealid) {
-              bid.dealId = sovrnBid.dealid;
-            }
-
-            bidmanager.addBidResponse(placementCode, bid);
-            impidsWithBidBack.push(id);
-          }
-        }
+    return {
+      method: 'POST',
+      url: `//ap.lijit.com/rtb/bid?src=${REPO_AND_VERSION}`,
+      data: JSON.stringify(sovrnBidReq),
+      options: {contentType: 'text/plain'}
+    };
+  },
+
+  /**
+   * Format Sovrn responses as Prebid bid responses
+   * @param {id, seatbid} sovrnResponse A successful response from Sovrn.
+   * @return {Bid[]} An array of formatted bids.
+  */
+  interpretResponse: function({id, seatbid}) {
+    let sovrnBidResponses = [];
+    if (id &&
+      seatbid &&
+      seatbid.length > 0 &&
+      seatbid[0].bid &&
+      seatbid[0].bid.length > 0) {
+      seatbid[0].bid.map(sovrnBid => {
+        sovrnBidResponses.push({
+          requestId: sovrnBid.impid,
+          bidderCode: spec.code,
```

Comment:
`bidderCode` will be set automatically by `bidderFactory` now, this line can be dropped